examples/temperature-measurement-app/telink/src/SensorManager.cpp

```diff
@@ -49,7 +49,7 @@ int16_t SensorManager::SensorEventHandler()
 #ifdef TEMPERATURE_SIMULATION_IS_USED
     static uint8_t nbOfRepetition = 0;
     static uint8_t simulatedIndex = 0;
-    if (simulatedIndex >= sizeof(mSimulatedTemp) - 1)
+    if (simulatedIndex >= sizeof(mSimulatedTemp) / sizeof(mSimulatedTemp[0]))
```

Comment:
Any reason this is not using ArraySize()?

Reply:
Fixed, didn't see this macro